tom/tom.py

```diff
@@ -1,10 +1,8 @@
 from twisted.words.protocols import irc
 from twisted.internet import reactor, protocol, defer
 from re import search, IGNORECASE
-from random import (
-                    choice,
-                    randint
-                    )
+from random import choice
+
 import random
```

Comment:
i feel like this is a redundant import. If you only need choice, then delete this line